DQMOffline/Trigger/plugins/GENERICDQM.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DQMOffline_Trigger_GENERICDQM_H
+#define DQMOffline_Trigger_GENERICDQM_H
+
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+struct MEbinning {
+  int nbins;
+  double xmin;
+  double xmax;
+};
+
+struct OBJME {
+  MonitorElement* numerator;
+  MonitorElement* denominator;
+  inline void clear(){
```

Comment:
Hi, it seemed I overlooked this comment, sorry. So no it is not done at the moment, but we can certainly change it for next time.